AzureCalling/Controllers/IntroViewController.swift

```diff
@@ -0,0 +1,262 @@
+//
```

Comment:
I don't see that we're locking orientation anywhere but I cannot rotate this screen to landscape mode.

Reply:
That code is unchanged from the `AppDelegate` file, line 24